apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -81,15 +81,21 @@ const ClassroomList = ({classrooms, onSelect, selectedId}) => classrooms.length
     <p>
       {locale.noClassroomsFound()}
     </p>
-    <a href="https://classroom.google.com/">
-      {locale.addRemoveGoogleClassrooms()}
-    </a>
+    {provider === 'google' ?
+      <a href="https://classroom.google.com/">
+        {locale.addRemoveGoogleClassrooms()}
+      </a> :
+      <a href="https://clever.com/">
+        {locale.addRemoveCleverClassrooms()}
+      </a>
+    }
```

Comment:
I do too, but then I accidentally nest them when adding code 🙁.  Thanks for catching!